<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimilarityMap</class>
 <widget class="QDockWidget" name="SimilarityMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>305</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFeatureMask</set>
  </property>
  <property name="windowTitle">
   <string>Similarity Map</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget_Container" native="true"/>
    </item>
    <item>
     <widget class="QFrame" name="frame_Controls">
      <property name="maximumSize">
       <size>
        <width>225</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton_LoadMap">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Load Map</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_WindowLower">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>211</width>
         <height>19</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_WindowUpper">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>211</width>
         <height>19</height>
        </rect>
       </property>
       <property name="value">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbShowMarkersInSpectrum">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>201</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Markers in Spectrum View</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
